src/Microsoft.ML.Data/DataLoadSave/Text/TextLoaderSaverCatalog.cs

```diff
@@ -11,33 +11,33 @@ namespace Microsoft.ML
     public static class TextLoaderSaverCatalog
     {
         /// <summary>
-        /// Create a text reader <see cref="TextLoader"/>.
+        /// Create a text loader <see cref="TextLoader"/>.
         /// </summary>
         /// <param name="catalog">The <see cref="DataOperations"/> catalog.</param>
         /// <param name="columns">The columns of the schema.</param>
         /// <param name="hasHeader">Whether the file has a header.</param>
         /// <param name="separatorChar">The character used as separator between data points in a row. By default the tab character is used as separator.</param>
         /// <param name="dataSample">The optional location of a data sample.</param>
-        public static TextLoader CreateTextReader(this DataOperations catalog,
+        public static TextLoader CreateTextLoader(this DataOperations catalog,
```

Comment:
>CreateTextLoader [](start = 33, length = 16)

My personal take on this is: TextReader is a much better name than TextLoader. I would rather rename the  TextLoader class to TextReader.. is that ok for the cmd usage.

Reply:
This is correct, we are replacing the `IDataLoader` idiom with the `IDataReader` idiom, as part of #581 and much subsequent work.

Reply:
`TextReader` is conflated with `System.IO.TextReader` and should be avoided. I've filed #2144 for renaming the class. I propose making the method name and return type consistent in this PR and revisiting the name when we rename the class.

cc: @glebuk @eerhardt

Reply:
Ah. Good point,. Maybe we should actually yname `IDataReader` to `IDataLoader`? And call the existing legacy `IDataLoader` interface something like `IDataLoaderLegacy`? What we all think?

Reply:
Similar renames to `IDataReaderEstimator` to I suppose `IDataLoaderEstimator`.

Reply:
My point is to be consistent between method's name and returned type, so either on of the following would be good for me: 👍 

public static Text**Loader** CreateText**Loader**()

public static Text**Reader** CreateText**Reader**()